scripts/azure-dev/AzureDev.Config.psm1

```
@@ -0,0 +1,516 @@
+    [ValidateSet(
+      'commit.gpgsign',
+      'gpg.format',
+      'user.email',
+      'user.name',
+      'user.signingkey'
+    )]
+    [string]$Key
+  )
+
+  if ($null -eq (Get-Command git -ErrorAction SilentlyContinue)) {
+    return ''
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-AzureDevLocalGitConfigValue' returns an object of type 'System.String' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)